pkg/server/keyclaim.go

```diff
@@ -60,7 +62,18 @@ func (s *keyClaimServlet) newKeyClaim(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	keyClaim, err := s.db.NewKeyClaim(region, originator)
+	hashId := vars["hashId"]
+
+	if hashId != "" {
```

Comment:
I've been pretty paranoid with most inputs; I would have verified the length here within some bounds (at least min 1 max 127–128 to match the db column, but probably there's a sensible minimum that makes more sense)

Reply:
Fixed in #57